From da8fc2ff6f2e9782b8d1a7b9c8ba6fe5f1242781 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 04 Jan 2022 16:56:38 +0200
Subject: [PATCH] [SW-69272] pci: update msix pba value

Currently, FW is writing 0x400 to mmPCIE_DBI_MSIX_PBA_OFFSET_REG, while
0x400 is the value that should have been written to PCI_MSIX_PBA_OFFSET
field inside that register.

Need to change the value in the MSIX PBA register to 0x2002.
(PCI_MSIX_PBA_OFFSET = 0x400, PCI_MSIX_PBA_BIR = 0x2)



Change-Id: Ibdb3efb58fa926788ef9661d53fab40e5a763c51
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 62db4c9..e0dddaf 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -709,7 +709,7 @@
 	}
 }
 
-#define MSIX_PBA_OFFSET_REG_VAL		0x400
+#define MSIX_PBA_OFFSET_REG_VAL		0x2002
 
 void set_pci_msix_pba_offset_val(void)
 {
